src/core/providers/arcgis/qgsarcgisportalutils.h

```diff
@@ -46,6 +47,8 @@ class CORE_EXPORT QgsArcGisPortalUtils
       ImageService, //!< ArcGIS Server image service
     };
 
+#ifndef SIP_RUN
```

Comment:
These could still be exposed couldn't they? Sip should handle it ok and call the correct method depending on whether a dict or the new class is passed for the argument

Reply:
No :sad: the sip/python can not be compiled do to the fact the functions have the same name but not the same definition. I may put different names but it looks uglier than hidding the new HttpHeader class.

Reply:
Hmm -- that's odd. We have plenty of other functions with the same number and different signatures, and sip is happy with those. What's the actual error you get?

Reply:
@nyalldawson 

Here are the errors I obtain if I remove the `#ifdef SIP_RUN`:
```
sip: core/auto_generated/providers/arcgis/qgsarcgisportalutils.sip:120: ::QgsArcGisPortalUtils::retrieveUserInfo() has overloaded functions with the same Python signature
```

Is that a bug or should I put back the `#ifdef SIP_RUN`?

Regards.